public/app/plugins/datasource/opentsdb/components/OpenTsdbDetails.tsx

```diff
@@ -53,6 +53,30 @@ export const OpenTsdbDetails = (props: Props) => {
           onChange={onInputChangeHandler('lookupLimit', value, onChange)}
         />
       </div>
+      <div className="gf-form-inline">
+        <div className="gf-form">
+          <FormField
+            labelWidth={7}
+            label="Min time interval"
+            inputEl={
+              <Input
+                className={'width-6'}
+                value={value.jsonData.interval || ''}
+                onChange={onInputChangeHandler('interval', value, onChange)}
+                placeholder="1m"
+                validationEvents={{
+                  [EventsWithValidation.onBlur]: [
+                    regexValidation(
+                      /^\d+(ms|[Mwdhmsy])$/,
```

Comment:
This will trigger if left empty. Again does not seem like we want to remove the old default of just leave it for the panel and make this mandatory.